cartoframes/examples.py

```diff
@@ -0,0 +1,5 @@
+from cartoframes import CartoContext
+
+def exampleContext():
+    """Returns a CartoContext corresponding to a CARTO account with example data"""
```

Comment:
We're going to need to write a full docstring on the usage of this

Reply:
I guess that this depends on hiding that it's a `CartoContext` or not. As I was proposing exposing it, it might not need documentation. What documentation were you missing?

Uhm, `CartoContext` hasn't any method for dataset discoverability within the account, has it? Maybe, instead of documenting the available datasets we could add listing of available datasets.